### PR TITLE
keystone: remove PKI format option for users

### PR DESCRIPTION
**Pull request by sidthe**
PKI tokens in Keystone are known to not be fully secure, we should not offer this option anymore

### CODE REVIEW

**Comment by sidthe** [comment]
[(3) Remove support for PKI tokens](https://trello.com/c/RJGh4Jg1/394-3-remove-support-for-pki-tokens)

File: crowbar_framework/app/helpers/barclamp/keystone_helper.rb [hunk 1]
```diff
@@ -31,7 +31,6 @@ def server_frontends_for_keystone(selected)
     def token_formats_for_keystone(selected)
       options_for_select(
         [
-          ["PKI", "PKI"],
```
Comment (Itxaka) [inline crowbar_framework/app/helpers/barclamp/keystone_helper.rb]: Do we still need this if there is only one option to choose? Should be removed IMHO

Comment (sidthe) [inline crowbar_framework/app/helpers/barclamp/keystone_helper.rb]: @Itxaka we still need this, as there is a plan to have fernet as an option as well

Comment (mjura) [inline crowbar_framework/app/helpers/barclamp/keystone_helper.rb]: If there is only one option, there is no need to select, choose option can be removed from Keystone proposal

**Comment by mjura** [comment]
In general looks ok, but as I said there is no need to keep select option in Keystone proposal only with one choice

**Comment by sidthe** [comment]
@mjura see above the same question from Itxaka and my reply to it. The plan is to have fernet introduced as an option very soon, so no need to drop the select.

**Comment by mjura** [comment]
@sidthe: ok please fix then comments in https://github.com/crowbar/crowbar-ha/blob/master/chef/cookbooks/crowbar-pacemaker/resources/sync_mark.rb#L32

**Comment by toabctl** [comment]
@sidthe what happens if somebody uses PKI and upgrades? Isn't there a migration needed to UUID?

**Comment by sidthe** [comment]
@toabctl we've looked at it with Itxaka, and we couldn't find anything that would hint to a possibility of such a migration. You have any ideas?

**Pull request merged**